.travis.yml

```
@@ -11,6 +11,7 @@ env:
     global:
         - SYMFONY_PHPUNIT_DIR=$HOME/.phpunit-bridge
         - SYMFONY_PHPUNIT_REMOVE='' # don't remove prophecy
+        - SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1
 
 php: [7.2, 7.1, 7.0, 5.6, 5.5, 5.4]
```

```suggestion
 php: [7.2, 7.1, 7.0, 5.6]
```

You're adding 5.4 and 5.5 runs via the matrix, so you don't need them here anymore.

---

thanks, I'll add your suggestion to the PR

---

Looks like travis is no longer able to parse the config file

---

Yes, there‘s now an extra whitespace before the `php` that does not belong there.

---

gotcha, ta